GameData/RealismOverhaul/RO_SuggestedMods/HabTech2/RO_HabTech2.cfg

```diff
@@ -1,157 +1,807 @@
+			amount = 1000
+			maxAmount = 1000
+		}
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 10000
+		}
+		TANK
+		{
+			name = Oxygen
+			amount = 10000
+			maxAmount = 10000
+		}
 		TANK
 		{
 			name = Food
+			amount = 350
+			maxAmount = 350
+		}
+		TANK
+		{
+			name = Waste
 			amount = 0
-			maxAmount = 8438.43
+			maxAmount = 250
+		}
+	}
+}
+
+@PART[ht2_moduleQuest]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = False
+	@rescaleFactor = 1.5625
+	@mass = 6.014
+	@title = ISS Quest Joint Airlock
+	@manufacturer = Boeing 
+	@description = The Quest Joint Airlock, previously known as the Joint Airlock Module, is the primary airlock for the International Space Station. Quest was designed to host spacewalks with both Extravehicular Mobility Unit (EMU) spacesuits and Orlan space suits. The airlock was launched on STS-104 on July 14, 2001. Before Quest was attached, Russian spacewalks using Orlan suits could only be done from the Zvezda service module and American spacewalks using EMUs were only possible when a Space Shuttle was docked. The arrival of Pirs docking compartment on September 16, 2001 provided another airlock from which Orlan spacewalks can be conducted.
+	@maxTemp = 1073.15 
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 20000
+		basemass = -1
+		type = ServiceModule
+		TANK
+		{
+			name = ElectricCharge
+			amount = 1000
+			maxAmount = 1000
 		}
-		
 		TANK
 		{
-			name = Water
+			name = CarbonDioxide
 			amount = 0
-			maxAmount = 5577.39
+			maxAmount = 10000
 		}
-		
 		TANK
 		{
 			name = Oxygen
+			amount = 100000
+			maxAmount = 100000
+		}
+	}
+
+}
+
+@PART[ht2_moduleUnity]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = False
+	@module = Part
+	%rescaleFactor = 1.5625
+	@category = Utility
+	@mass = 11.312
+	@title = ISS Unity (Node 1)
+	@manufacturer = Boeing
+	@description = The Unity connecting module was the first U.S.-built component of the International Space Station. It is cylindrical in shape, with six berthing locations (forward, aft, port, starboard, zenith, and nadir) facilitating connections to other modules. Unity was built for NASA by Boeing in a manufacturing facility at the Marshall Space Flight Center in Huntsville, Alabama. Sometimes referred to as Node 1, Unity was the first of the three connecting modules; the other two are Harmony and Tranquility.
+	@maxTemp = 1073.15
+	@CrewCapacity = 1
+	@vesselType = Station
+	!RESOURCE,* {}
+	!MODULE[ModuleCommand] {}
+	!MODULE[ModuleFuelTanks],* {}
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 200
+		basemass = -1
+		type = ServiceModule
+		TANK
+		{
+			name = ElectricCharge
+			amount = 200000
+			maxAmount = 200000
+		}
+		TANK
+		{
+			name = CarbonDioxide
 			amount = 0
-			maxAmount = 772590
+			maxAmount = 10000
 		}
-		
 		TANK
 		{
-			name = Waste
+			name = Oxygen
+			amount = 10000
+			maxAmount = 10000
+		}
+	}
+}
+
+@PART[ht2_radialTrussPort]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = False
+	!rescaleFactor{}
```

Comment:
Syntax: `!rescaleFactor,* = DEL`

Reply:
> Syntax: `!rescaleFactor,* = DEL`

I'm in the process of making the required changes, could you please just confirm the issue with duplicate rescaleFactor instructions in the original file. will !rescaleFactor,* = DEL get rid of ALL rescaleFactor instructions within the configuration file or just the first one it comes across?

either way, in the configs for the affected parts I need to have a '@rescaleFactor = 1.5625 if it just deletes one, or %rescaleFactor = 1.5625 if it removes them both.

Reply:
`!rescaleFactor,* = DEL` will remove all instances of `rescaleFactor`.  Deleting all, and then creating a new one with `%rescaleFactor` will provide what we want, and will work even after the original mod fixes its error, if/when that happens.

Note that `%` is "create or edit."  If one doesn't exist, it'll create.  If one exists, it'll edit.  If multiple exists, it'll edit the first one.

Reply:
cool, that's why I wrote it like I did in the first instance because I was trying to avoid deleting all rescaleFactor instructions as I still needed one. However I will alter it to how you have suggested.